release/nightly_tests/dask_on_ray/dask_on_ray_app_config.yaml

```diff
@@ -13,6 +13,7 @@ python:
 post_build_cmds:
   # - pip install fastparquet
   - pip3 install boto3 s3fs
+  - pip3 install -U s3fs
```

Comment:
Can you add to this file as well https://github.com/ray-project/ray/blob/master/release/ray_release/byod/byod_dask_test.sh#L9? This is for the BYOD version of the test. Thankkks